playbook-website/app/javascript/components/Website/src/layouts/Sidebar/NavComponents/OtherNavComponent.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
```

Comment:
What is an OtherNavComponent? Can we describe this file better?

Reply:
This isn't being used, and was copied over from Nida's work. I'll go ahead and remove this until we implement the react sidebar. For now, per your request, we are referencing the rails version that is rendered via her `MainSidebar` folder still, so that all the links redirect users to the legacy views.